common/config/src/lib.rs

```diff
@@ -452,13 +442,16 @@ fn print_settings(yaml: &str, config_path: &Path) {
     std::process::exit(0);
 }
 
-fn parse_k8s_tracking_or_warn(
+fn parse_k8s_enum_config_or_warn<T: FromStr + std::fmt::Display + std::fmt::Debug>(
```

Comment:
Refactored this helper function to take a generic type so I could use it for both `K8sTrackingConf` and `K8sLeaseConf`.